Things.md

```diff
@@ -169,4 +170,12 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing Classes
+
+A Thing Class is the definition of a single type of Thing (i.e. earlier we used CastleFireball). Each unique in-game Thing has a Thing Class defining all of its properties. There can, of course, be multiple instances of a single Thing Class.
```

Comment:
Overall, this paragraph isn't very informative. 

The goal here is to inform the reader that Things follow a class-based inheritance driven by ObjectMakr. You're partially defining what classes are but not elaborating, so the end result is confusion.